lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -274,8 +263,7 @@ sub _exceeds_limit ($self, $needed) {
     return 0;
 }
 
-sub _check_limits {
-    my ($self, $needed, $to_preserve) = @_;
+sub _check_limits ($self, $needed, $to_preserve) {
```

Comment:
```suggestion
sub _check_limits ($self, $needed, $to_preserve = undef) {
```

Because it you look for the use of this variable you will see that its existence and truthiness is checked

Reply:
$to_preserve = undef means that it has a default value of undef, right?

Reply:
Yes